lib/vagrant-service-manager/command.rb

```diff
@@ -180,9 +180,9 @@ def restart_service(service)
         command = if SCCLI_SERVICES.include? service
                     # TODO : Handle the case where user wants to pass extra arguments
                     # to OpenShift service
-                    "sccli #{service}"
+                    "sccli #{service} #{operation}"
                   else
-                    "systemctl restart #{service}"
+                    "systemctl #{operation} #{service}"
```

Comment:
-1 for this part. The service-manager is not a proxy to systemctl. How do you explain cohesively the term "service" in the context of vagrant-service-manager when we are including systemd services in this command. How come I can start/stop/restart systemd commands via the plugin, but not get the status of any of these services?

What is wrong with:

    systemctl start cockpit

I am still missing the use case here.